build-browser.js

```diff
@@ -0,0 +1,23 @@
+const pkg = require('./package.json');
+const rollup = require('rollup').rollup;
+const babel = require('rollup-plugin-babel');
+
+rollup({
+  entry: 'src/index.js',
+  plugins: [
+    babel({
+      presets: ['es2015-rollup'],
+      plugins: ['transform-node-env-inline'],
+      babelrc: false,
+    }),
+  ],
+}).then(bundle => (
+  bundle.write({
+    dest: 'validator.js',
+    format: 'umd',
+    moduleName: pkg.name,
```

Comment:
@kevinansfield the `rollup` package is responsible for generating the output code. There may be a rollup plugin that works for Ember, or a patch to the main `rollup` repo might be required. I'm happy to quickly accept a PR that fixes the problem or updates dependencies. I don't use Ember so cannot test fixes.